superchain/configs/sepolia/boba.yaml

```diff
@@ -0,0 +1,22 @@
+name: boba
+chain_id: 28882
+public_rpc: https://sepolia.boba.network
+sequencer_rpc: https://sepolia.boba.network
+explorer: https://bobascan.com
```

Comment:
I think this should be https://testnet.bobascan.com ?

Reply:
Fixed